cmd/wallet-web/src/components/DIDManagement/Preferences.vue

```diff
@@ -55,18 +55,34 @@
       </div>
     </div>
 
-    <button class="mt-5 btn-primary" :disabled="!preferencesChanged" @click="updatePreferences">
-      Update Preferences
-    </button>
+    <styled-button
+      id="update-btn"
+      class="mt-5"
+      :disabled="!preferencesChanged"
+      :loading="loading"
+      type="primary"
+      @click.native="updatePreferences()"
+    >
+      <span>Update Preferences</span>
+    </styled-button>
+
+    <div v-if="updateSuccessful && !preferencesChanged">
+      <label id="save-anydid-success">Saved your DID successfully.</label>
```

Comment:
This is update preference operation (not Save DID). probably we can update the success message to something like `Preferences updated successfully`

Reply:
Right. Forgot to update the text 😅